src/server.rs

```diff
@@ -183,6 +183,13 @@ fn delete_host<S: Storage>(s: S, name: ServiceName, ip: String, port_string: Str
         Err(_e) => return res_400(format!("Given port is invalid as integer: {}", port_string)),
     };
 
+    match s.get_item(&name, &ip, &port) {
```

Comment:
This could have race condition when multiple requests are sent simultaneously.
It would be better if it uses ReturnValues to know the result of deletion.
https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_DeleteItem.html#DDB-DeleteItem-request-ReturnValues